README.md

```diff
@@ -288,6 +289,28 @@ spectator.mouse.dblclick('.selector');
 
 Note that each one of the above methods will also run `detectChanges()`.
 
+### OnInit lifecycle tests
```

Comment:
I think it needs to be in the `Mocking Providers` section.

Reply:
sounds good. I'll move it.

Reply:
done. changed the title to "Mocking OnInit dependencies" as well. Hopefully that makes sense.